### ISSUE TITLE
Add support for multiple consecutive character replacements

### ISSUE THREAD

**Issue by hplewis** [hplewis/emoji-slack#3]
Some sort of look-ahead or prioritize longer character replacements. Might require a probability they are applied so that the individual character replacements are still possible.

**Comment by TennisGazelle** [comment]
not necessarily, we can just take care of those before we do the single char replacements

**Comment by hplewis** [comment]
True. Just so long as replacements have a roughly equal probability of happening.
